Template/Tag/EtrackerTag.php

```diff
@@ -121,6 +126,129 @@ public function getParameters()
                 $field->description = 'The event\'s type Name, for example an value of a send form...';
                 $field->customUiControlTemplateFile = self::FIELD_TEMPLATE_VARIABLE;
```

Comment:
@ben-dw This has changed, and I am afraid you need to make the similar change in your code
Refer: https://github.com/matomo-org/tag-manager/blob/4.x-dev/Template/Tag/EtrackerTag.php#L41